esp-wifi/src/wifi/mod.rs

```diff
@@ -2697,6 +2699,7 @@ mod embedded_svc_compat {
         }
     }
 
+    #[allow(clippy::from_over_into)]
```

Comment:
This was a bit too much work for this PR, we should file another issue for these.

Reply:
Jk, wasn't too hard